huggingface/pytorch/inference/docker/1.10/py3/Dockerfile.cpu.os_scan_allowlist.json

```diff
@@ -2,8 +2,8 @@
    "rdflib":[
       {
          "description":"[rdflib](https://pypi.org/project/rdflib/) is a Python library for working with RDF, a simple yet powerful language for representing information.\n\nAffected versions of this package are vulnerable to Server-Side Request Forgery (SSRF). `rdflib` provides no way to control how external references are resolved, nor a way to implement caching of external resources.\r\n\r\nIf a web service takes `POST`ed `JSON-LD` data, `rdflib` will attempt to resolve any URL in the `@context`.\r\nThis can lead to:\r\n1. attackers being able to probe internal networks, by having `rdflib` request potential non-public URLs\r\n2. reflection attacks, if the same or slightly-different URLs are repeated multiple times in the `@context`\r\n3. resource exhaustion, as the entire remote file is loaded into memory before JSON parsing is attempted\r\n4. Denial of Service, if web or task workers are tied up waiting for extended periods for HTTP requests to complete\r\n5. attackers being able to probe the local filesystem using `file://` URLs",
-         "vulnerability_id":"IN1-PYTHON-RDFLIB-1324490",
-         "name":"IN1-PYTHON-RDFLIB-1324490",
+         "vulnerability_id":"SNYK-PYTHON-RDFLIB-1324490",
```

Comment:
was this incorrectly named before, or did this change?

Reply:
I think this changed on the Scanner side itself. Probably they named it incorrectly earlier, or took some decision to change the ids.